src/bp/core/services/dialog/prompt-manager.ts

```diff
@@ -84,7 +84,7 @@ export class PromptManager {
     const { minConfidence, prompt } = this._getPrompt(node, event)
 
     const extractedVars = await this.evaluateEventVariables(events, prompt)
-    const highest = _.orderBy(extractedVars, 'confidence', 'desc')[0]
+    const highest = _.orderBy(extractedVars, 'confidence', 'desc')[0] ?? { confidence: 0, extracted: false }
```

Comment:
I think we could change the name of the property, because extracted is actually the value that was extracted, and should be undefined